source/includes/examples/_energyplus.html.md

```diff
@@ -2,7 +2,7 @@
 
 > ![EnergyPlus Graph](../images/energyplus-graph.png)
 
-[**Download the full source code on GitHub**][1] so you can run it yourself on the Bonsai Platform.
+[**Download the full source code on GitHub**][1] if you want to run this simulator locally or create a new BRAIN on [beta.bons.ai][4] to run it yourself on the Bonsai Platform.
```

Comment:
@codemonkey9000 looks like this comment didn't get updated. Can you phrase it the same as the Cart Pole comment?